Makefile

```diff
@@ -19,13 +19,21 @@ NODE ?= node
 NPM ?= npm
 BUNDLER ?= $(BIN)/bundler
 I18N_CALYPSO ?= $(NODE_BIN)/i18n-calypso
-SASS ?= $(NODE_BIN)/node-sass --include-path 'client'
+SASS ?= $(NODE_BIN)/node-sass --include-path 'client' --include-path 'build'
 RTLCSS ?= $(NODE_BIN)/rtlcss
 AUTOPREFIXER ?= $(NODE_BIN)/postcss -r --use autoprefixer --autoprefixer.browsers "last 2 versions, > 1%, Safari >= 8, iOS >= 8, Firefox ESR, Opera 12.1"
 RECORD_ENV ?= $(BIN)/record-env
 ALL_DEVDOCS_JS ?= server/devdocs/bin/generate-devdocs-index
 COMPONENTS_USAGE_STATS_JS ?= server/devdocs/bin/generate-components-usage-stats.js
 
+# hash command
```

Comment:
The hash file is used to remember the set of sass files. With a naive rule that just depends on the result of finding all sass files, if a file was deleted, the rule wouldn't rerun, because the rule now depends on fewer files than it did before, so everything looks up-to-date.

md5sum is part of GNU coreutils so it should be on all the linuxes. I also checked that it is available under msys2